catroid/src/main/java/org/catrobat/catroid/common/LookData.java

```diff
@@ -75,6 +75,8 @@ public class LookData implements Cloneable, Nameable, Serializable {
 
 	private boolean isWebRequest = false;
 
+	private boolean isCopy = false;
```

Comment:
I saw that this flag is only used in your unit tests. Maybe you can also achieve the same testing without this flag, then we wouldn't need these changes here.